genie_registry/clinical.py

```diff
@@ -956,6 +1009,16 @@ def _validate(self, clinicaldf):
                 )
         else:
             total_error.write("Patient Clinical File: Must have DEAD column.\n")
+
+        # CHECK: INT DOD against INT CONTACT
+        has_int_dod_and_contact = process_functions.checkColExist(
+            clinicaldf, ["INT_DOD", "INT_CONTACT"]
+        )
+        if has_int_dod_and_contact:
+            invalid_int_dod_indices = _check_int_dod_validity(clinicaldf)
+            errors, warnings = _check_int_dod_validity_message(invalid_int_dod_indices)
```

Comment:
Nit:
`errors, _`
As warnings is not used

Reply:
This is adapted from https://github.com/Sage-Bionetworks/Genie/blob/3f318022e1ef3e5de8c774ce79f7b521ab5653b4/genie_registry/clinical.py#L475-L535 and we might need to output warnings in the future.